src/CreateGuesser.tsx

```diff
@@ -77,10 +80,16 @@ export const IntrospectedCreateGuesser = ({
     displayOverrideCode(getOverrideCode(schema, writableFields));
   }
 
-  const hasFileField = inputChildren.some(
-    (child) =>
-      typeof child === 'object' && 'type' in child && child.type === FileInput,
-  );
+  const hasFileFieldElement = (elements: Array<ReactNode>): boolean =>
```

Comment:
Shouldn't this code be in the PR https://github.com/api-platform/admin/pull/475?

Reply:
Nope, multiple file upload does not place  FileInput element deeper in the tree, it is still on the first level just with additional `multiple` attribute, so this works as it is:
```
  <CreateGuesser {...props} >
      <FileInput source={"files"} multiple={true} >
   (...)
```
But tabbed form place it deeper, so the there is a need to look deeper: 
```
  <CreateGuesser {...props} >
    <FormTab lable={"files"}>
      <FileInput source={"files"}  >
   (...)
```
And after all looking deeper it support any other nesting of FileInput component.